install.js

```diff
@@ -138,12 +138,12 @@ var configItems = [
     description: "Sender email address",
     default: ''
   },
```

Comment:
This might be a good opportunity to remove the prompt for SMTP settings also - it's not really required for most installs, at present.